org-formation/700-aws-sso/_tasks.yaml

```diff
@@ -519,51 +519,72 @@ SsoApplicationManager:
+      - Name: SynapseAthenaUserAccessPolicy
+  StackName: !Sub '${resourcePrefix}-${appName}-synapse-dw-athena-user'
+  StackDescription: 'Access to Synapse Data Warehouse via AWS Athena'
+  TerminationProtection: false
+  DefaultOrganizationBindingRegion: !Ref primaryRegion
+  DefaultOrganizationBinding:
+    IncludeMasterAccount: true
+  OrganizationBindings:
+  Parameters:
+    instanceArn: !Ref instanceArn
+    principalId: !Ref adminGroup # by default we admit admin's.  Below we 'inherit' this stack for specific user groups
+    permissionSetName: 'synapse-dw-athena'
+    sessionDuration: 'PT12H'
+    masterAccountId: !Ref MasterAccount
+
 # Role for a user that can only access AWS Athena in the Synapse Dev account
-SsoSynapseDevAthenaUser:
+SsoSynapseDWDevAthenaUser:
   Type: update-stacks
+  DependsOn: SsoSynapseDWAthenaUser
   Template: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.5.1/templates/SSO/aws-sso.njk
   TemplatingContext:
     customerManagedPolicies:
       - Name: SynapseAthenaUserAccessPolicy
-  StackName: !Sub '${resourcePrefix}-${appName}-synapse-dev-athena-user'
+  StackName: !Sub '${resourcePrefix}-${appName}-synapse-dw-dev-athena-user'
   StackDescription: 'Access to AWS Athena in Synapse Dev account'
   TerminationProtection: false
   DefaultOrganizationBindingRegion: !Ref primaryRegion
   DefaultOrganizationBinding:
-    IncludeMasterAccount: true
+    IncludeMasterAccount: false
   OrganizationBindings:
     TargetBinding:
       Account: !Ref SynapseDevAccount
   Parameters:
     instanceArn: !Ref instanceArn
     principalId: !Ref synapseDevAthenaGroup
-    permissionSetName: 'athena-user-dev'
+    permissionSetArn: !CopyValue [ !Sub '${resourcePrefix}-${appName}-synapse-dw-athena-permission-set-arn' ]
     sessionDuration: 'PT12H'
-    masterAccountId: !Ref MasterAccount
 
 # Role for a user that can only access AWS Athena in the Synapse Prod account
-SsoSynapseProdAthenaUser:
+SsoSynapseDWProdAthenaUser:
   Type: update-stacks
+  DependsOn: SsoSynapseDWAthenaUser
   Template: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.5.1/templates/SSO/aws-sso.njk
   TemplatingContext:
     customerManagedPolicies:
       - Name: SynapseAthenaUserAccessPolicy
-  StackName: !Sub '${resourcePrefix}-${appName}-synapse-prod-athena-user'
+  StackName: !Sub '${resourcePrefix}-${appName}-synapse-dw-prod-athena-user'
   StackDescription: 'Access to AWS Athena in Synapse Prod account'
   TerminationProtection: false
   DefaultOrganizationBindingRegion: !Ref primaryRegion
   DefaultOrganizationBinding:
-    IncludeMasterAccount: true
+    IncludeMasterAccount: false
   OrganizationBindings:
     TargetBinding:
       Account: !Ref SynapseProdAccount
   Parameters:
     instanceArn: !Ref instanceArn
     principalId: !Ref synapseProdAthenaGroup
-    permissionSetName: 'athena-user-prod'
+    permissionSetArn: !CopyValue [ !Sub '${resourcePrefix}-${appName}-synapse-dw-athena-permission-set-arn' ]
```

Comment:
this should be `permissionSetArn: !CopyValue [ !Sub '${resourcePrefix}-${appName}-synapse-dw-athena-user-permission-set-arn' ]`